www/javascript/site-content-slider.js

```diff
@@ -18,11 +18,13 @@
  *  if left without height, the pages' height will be set to the height
  *  of the tallest page.
  */
+
```

Comment:
This extra whitespace disconnects the documentation from the method it's documenting.